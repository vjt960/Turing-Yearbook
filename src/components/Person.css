article {
  height: 100%;
  width: 100%;
  border-radius: 5%;
  display: flex;
  flex-direction: column;
  align-items: center;
  background-color: rgba(255, 255, 255, 0.95);
}

img {
  object-fit: cover;
  height: 255px;
  width: 250px;
  margin: 5% 0 0 0;
  border-radius: 5%;
}

.text {
  margin: 5px;
  width: 250px;
  font-size: 15px;
}

.name {
  color: hsl(9, 100%, 64%)
}

.super {
  color: hsl(218, 78%, 67%);
}

