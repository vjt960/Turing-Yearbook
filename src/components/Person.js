import React from 'react';
import './Person.css';

const Person = ({ image, name, quote, superlative }) => {
  return (
    <article className='person-card'>
      <img src={image} />
      <h3 className='text name'>{name}</h3>
      <i className='text quote'>{quote}</i>
      <b className='text super'>{superlative}</b>
    </article>
  )
}

export default Person;
