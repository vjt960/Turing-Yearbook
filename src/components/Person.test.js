import React from 'react';
import { shallow } from 'enzyme';
import Person from './Person.js';

describe('Person', () => {
  it('works', () => {
    const person = {id: 1, name: 'Eric', quote: 'You miss all the shots you don\'t take.', superlative: 'Most Likely to Win the PGA', photo: './eric.jpg'};
  const wrapper = shallow(
    <Person 
      image={person.photo} 
      name={person.name} 
      quote={person.quote} 
      superlative={person.superlative} 
      key={0}
    />
  );

  expect(wrapper).toMatchSnapshot();
  })
});