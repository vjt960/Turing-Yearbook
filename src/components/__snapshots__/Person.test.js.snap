// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `ShallowWrapper {}`;

exports[`Person works 1`] = `ShallowWrapper {}`;
